mappings/net/minecraft/client/render/WorldRenderer.mapping

```diff
@@ -24,7 +24,8 @@ CLASS dkm net/minecraft/client/render/WorldRenderer
 	FIELD X chunkRendererList Ldkc;
 	FIELD Y renderDistance I
 	FIELD a DIRECTIONS [Lfa;
-	FIELD aa renderedEntities I
+	FIELD aa regularEntityCount I
```

Comment:
Do we need the `regular`? I feel like it goes without saying

Reply:
Maybe, but I just named it like that to match the text: `Regular: x/y Block: z/w`.

Reply:
Ah, didn't realise it was named like that in-game. I suppose either works fine

Reply:
It excludes global entities (currently only the lightning, I think), which are normally considered entities in other parts of the code.